src/components/ModalConfirmTransfer.tsx

```diff
@@ -64,7 +64,7 @@ export default function ModalConfirmTransfer({ closeModal }: ModalProps) {
     >
       <DModalHeader
         showCloseButton
-        onClose={() => closeModal()}
+        onClose={() => closePortal()}
```

Comment:
Las mismas observaciones que Eduard
```suggestion
        onClose={closePortal}
```